source/install/macos-source.rst

```diff
@@ -3,12 +3,11 @@ macOS 编译安装
 
 .. attention::
 
-   ARM 架构的 Mac 可以用 SAC 二进制文件直接安装使用，但源码无法直接编译通过。
-   另外，xquartz 需要到官网手动下载，Homebrew 安装的不能使用。
+   SAC 源代码年代久远，与现行的语法不相容。所以，实际上 macOS 上不能编译 SAC ，无解决办法。
```

Comment:
I don't think so. SAC can be compiled on macOS.

Reply:
你可以试试看啊

Reply:
It definitely can be compiled on macOS, otherwise IRIS won't be able to provide the binary package for IRIS. The only issue is that we don't know how IRIS compiles the binary and what flags they pass to the compilers.

Reply:
Let's keep this PR open. We could contact IRIS for this issue when any of us are available.

Reply:
他编译得早。那时候，编译器允许那些写法

Reply:
现在不能编译就是不能编译啊

Reply:
我觉得 @seisman 说的是合理的，现在只是我们不能编译。IRIS 当然是可以编译的，要不然就不会编译出可执行程序直接给我们用了。

Reply:
其实不用纠结这个，因为102能编译，102合并过来后啥都没有了

Reply:
> 我觉得 @seisman 说的是合理的，现在只是我们不能编译。IRIS 当然是可以编译的，要不然就不会编译出可执行程序直接给我们用了。

他编译的时候好像是2013年，也许那时候编译器还能接受sac的写法，所以他能编译。你可以试试现在能不能编译，看看报错怎么说的